### PR TITLE
Host Isolation 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses #742.

Preview here.

### CODE REVIEW

**Review by kevinlog — APPROVED** [review]
LGTM!

**Review by caitlinbetz — APPROVED** [review]

File: docs/management/admin/host-isolation-ov [hunk 1]
```diff
@@ -0,0 +1,69 @@
+[[host-isolation-ov]]
+[chapter, role="xpack"]
+= Host isolation
+
+Host isolation allows you to isolate Windows and macOS hosts from your network, blocking communication with any other hosts on your network until the host is released. Isolating a host is useful when responding to malicious activity or to prevent potential attacks, as it prevents lateral movement across other hosts.
```
Comment (caitlinbetz) [inline docs/management/admin/host-isolation-ov:5]: only thing i can think to add is that isolated hosts will still be able to communicate with kibana/ ES

File: docs/management/admin/host-isolation-ov [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+All actions executed on a host are tracked in the host’s activity log, which you can access from the Endpoints page. See <<view-host-isolation-details, View host isolation details>> for more information.
+
+[discrete]
+[[isolate-a-host]]
+== Isolate a host
+
+To isolate a host from a case alert:
+
+. Go to *Investigate -> Cases*, then select  and click on the appropriate case to view the case activity. Ensure you are viewing a case with at least one alert attached to it.
```
Comment (joepeeples) [inline docs/management/admin/host-isolation-ov:24]: Probably doesn't affect the final rendering, but there's an extra space after "select".
```suggestion
. Go to *Investigate -> Cases*, then select and click on the appropriate case to view the case activity. Ensure you are viewing a case with at least one alert attached to it.
```

File: docs/management/admin/host-isolation-ov [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+. Find the appropriate alert, then click the *Show alert details* button *(>)*. The alert details flyout opens.
+. Click *Take action -> Isolate*.
+. Enter a comment describing why you’re isolating the host (optional).
+. Click *Confirm*.
+
+To isolate a host from an endpoint:
+
+. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout appears.
```
Comment (joepeeples) [inline docs/management/admin/host-isolation-ov:32]: Throughout the file, some flyouts "appear" and some "open." I think they're both correct, but maybe just use a single term for consistency? My personal preference is "opens," so I edited for that throughout, but you could also change all the "opens" to "appears."
```suggestion
. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout opens.
```

File: docs/management/admin/host-isolation-ov [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+To release a host from a case alert:
+
+. Go to *Investigate -> Cases*, then and click on the appropriate case to view the case activity its details.
+. Find the appropriate alert, then and click the *Show alert details* button *(>)*. The alert details flyout opens.
+. From the alert details flyout, click *Take action > Release host*.
+
+To release a host from an endpoint:
+
+. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout appears.
```
Comment (joepeeples) [inline docs/management/admin/host-isolation-ov:52]:
```suggestion
. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout opens.
```

File: docs/management/admin/host-isolation-ov [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+[discrete]
+[[view-host-isolation-details]]
+== View host isolation details
+
+The host activity log tracks all actions performed on the host, including who made the host isolation request, when they made it, comments they provided, and when the host received the request to isolate.
+
+To view the host’s isolation details:
+
+. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout appears.
```
Comment (joepeeples) [inline docs/management/admin/host-isolation-ov:65]:
```suggestion
. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout opens.
```

File: docs/management/admin/host-isolation-ov [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+After the host is successfully isolated, an *Isolated* status is added next to the *Agent Status*.
+
+[discrete]
+[[release-a-host]]
+== Release a host
+
+To release a host from a case alert:
+
+. Go to *Investigate -> Cases*, then and click on the appropriate case to view the case activity its details.
```
Comment (joepeeples) [inline docs/management/admin/host-isolation-ov:46]:
```suggestion
. Go to *Investigate -> Cases*, then click on the appropriate case to view the case activity its details.
```

File: docs/management/admin/host-isolation-ov [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+. Find the appropriate alert, then and click the *Show alert details* button *(>)*. The alert details flyout opens.
```
Comment (joepeeples) [inline docs/management/admin/host-isolation-ov:47]:
```suggestion
. Find the appropriate alert, then click the *Show alert details* button *(>)*. The alert details flyout opens.
```

File: docs/management/admin/host-isolation-ov [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+Host isolation allows you to isolate Windows and macOS hosts from your network, blocking communication with any other hosts on your network until the host is released. Isolating a host is useful when responding to malicious activity or to prevent potential attacks, as it prevents lateral movement across other hosts.
+
+NOTE: To isolate and release hosts, you must have the built-in superuser role. For more information, see {ref}/built-in-users.html[Built-in users].
+
+[role="screenshot"]
+image::images/isolated-host.png[Shows a host that's been isolated]
+
+You can isolate a host via an alert attached to a case or from the Endpoints list. Once a host is successfully isolated, an *Isolated* status displays next to the *Agent Status* field, which you can view on the alert details flyout or Endpoints list table.
+
+TIP: If the request fails, verify that the Agent and your Endpoint are both online before trying again.
```
Comment (joepeeples) [inline docs/management/admin/host-isolation-ov:14]: Lowercase if not referring to something labeled "Endpoint"?
```suggestion
TIP: If the request fails, verify that the Agent and your endpoint are both online before trying again.
```

**Review by joepeeples — COMMENTED** [review]
A few comments and edits.

File: docs/management/admin/host-isolation-ov [hunk 9]
```diff
@@ -0,0 +1,69 @@
+[[host-isolation-ov]]
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/management/admin/host-isolation-ov:1]: Yeah, I tried adding the extension when I initially created the file, but it didn't take for some reason. @jmikell821 should I rename this file in Atom to add the extension?

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 10]
```diff
@@ -0,0 +1,69 @@
+[[host-isolation-ov]]
+[chapter, role="xpack"]
+= Host isolation
+
+Host isolation allows you to isolate Windows and macOS hosts from your network, blocking communication with any other hosts on your network until the host is released. Isolating a host is useful for responding to malicious activity or preventing potential attacks, as it prevents lateral movement across other hosts. After hosts are isolated, they can still send data to {es} and {kib}.
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-ov.asciidoc:5]:
```suggestion
Host isolation allows you to isolate Windows and macOS hosts from your network, blocking communication with any other hosts on your network until the host is released. Isolating a host is useful for responding to malicious activity or preventing potential attacks, as it prevents lateral movement across other hosts. Isolated hosts can still send data to {es} and {kib}.
```

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+NOTE: To isolate and release hosts, you must have the built-in superuser role. For more information, see {ref}/built-in-users.html[Built-in users].
+
+[role="screenshot"]
+image::images/isolated-host.png[Shows a host that's been isolated]
+
+You can isolate a host via an alert attached to a case or from the Endpoints list. Once a host is successfully isolated, an *Isolated* status displays next to the *Agent Status* field, which you can view on the alert details flyout or Endpoints list table.
+
+TIP: If the request fails, verify that the Agent and your endpoint are both online before trying again.
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-ov.asciidoc:14]: Still not 100% on case, but let's lowercase it for now. Its proper noun `Elastic Agent` {agent} is title case. 
```suggestion
TIP: If the request fails, verify that the agent and your endpoint are both online before trying again.
```

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+. Go to *Investigate -> Cases*, then click on the appropriate case to view the case activity its details.
+. Find the appropriate alert, then click the *Show alert details* button *(>)*. The alert details flyout opens.
+. From the alert details flyout, click *Take action > Release host*.
+
+To release a host from an endpoint:
+
+. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout opens.
+. Click *Take action -> Release host*.
+
+After the host is successfully released, the *Isolated* status is removed from the *Agent Status*.
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-ov.asciidoc:55]:
```suggestion
After the host is successfully released, the `Isolated` status is removed from the *Agent Status*.
```

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+[discrete]
+[[view-host-isolation-details]]
+== View host isolation details
+
+The host activity log tracks all actions performed on the host, including who made the host isolation request, when they made it, comments they provided, and when the host received the request to isolate.
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-ov.asciidoc:61]:
```suggestion
The host activity log tracks all actions performed on the host, including who made the host isolation request and when, comments added, and when the host received the request to isolate.
```

**Review by jmikell821 — COMMENTED** [review]
A few small comments but other than that, LGTM. You can merge/backport once these are added, and any last-minute edits we can add in a subsequent PR. Thank you!

**Pull request merged**